frontend/src/metabase/query_builder/components/ActionsWidget.jsx

```diff
@@ -88,7 +88,11 @@ export default class ActionsWidget extends Component {
         const action = getModeActions(mode, card, tableMetadata)[index];
         if (action && action.popover) {
             this.setState({ selectedActionIndex: index });
-        } else if (action && action.card) {
+        } else if (action && action.url) {
+          const nextUrl = action.url();
+          this.props.onChangeLocation(nextUrl);
```

Comment:
It's somehow already in the props. Eg I tested this manually and it works.

I don't know where it's being added, but I thought it was a bulk add of a bunch of actions somewhere up the stack.